tasks/create-github-release/README.md

```diff
@@ -14,6 +14,9 @@ a `release` dir.
 | release_version | The version string to use creating the release | No | - |
 | content_directory | The directory inside the workspace to find files for release | No | - |
 
+## Changes since 1.0.2
```

Comment:
`s/since/in/`

Reply:
Should the previous versions Changes remain which 'since' ?

Reply:
It is fine to just change the new entry. We didn't go through and change the old ones on other tasks/pipelines